### DISCUSSION TITLE
Invalid partials/Missing signage points since 2.1.2 update

### DISCUSSION THREAD

**Discussion by wobblewoo** [Chia-Network/chia-blockchain#17091] [Support]
Hello.

On updating to 2.1.2 i got thousands of Missing signage points reported in the GUI (Ubuntu 22.04.3 LTS) and alerts from SpacePool 'Invalid Partials'.

The errors disappear if i then downgrade back to 2.1.1. Same machine, same disks, same plots (uncompressed) and same wired internet connection. 

I did all the checks on: https://docs.chia.net/faq/#why-does-my-chia-farm-say-missing-signage-points

I'm not sure if i should send log files but ive got alot of entries like these two:
`2023-12-19T13:25:36.192 full_node chia.full_node.full_node: WARNING  Transaction queue full for peer 1234567891011121314151617181912021222324252627282930313233343536
2023-12-19T13:25:36.192 full_node chia.full_node.full_node: WARNING  Transaction queue full for peer 1234567891011121314151617181912021222324252627282930313233343536
2023-12-19T13:25:36.192 full_node chia.full_node.full_node: WARNING  Transaction queue full for peer 1234567891011121314151617181912021222324252627282930313233343536`

`2023-12-19T12:05:32.057 full_node chia.full_node.full_node: INFO     End of slot not added CC challenge 1234567891011121314151617181912021222324252627282930313233343536
2023-12-19T12:05:32.064 full_node chia.full_node.full_node_store: INFO     Don't have challenge hash 1234567891011121314151617181912021222324252627282930313233343536, caching EOS
2023-12-19T12:05:32.065 full_node chia.full_node.full_node: INFO     End of slot not added CC challenge 1234567891011121314151617181912021222324252627282930313233343536
2023-12-19T12:05:32.065 full_node chia.full_node.full_node_store: INFO     Don't have challenge hash 1234567891011121314151617181912021222324252627282930313233343536, caching EOS`

If relevant, the machine is an Intel i5-3470 CPU @ 3.20GHz x4 - 8Gb RAM - 512Gb SSD - Built in graphics.

Many thanks.

**Comment by BrandtH22** [comment]
Hey @wobblewoo , the missing signage points is much more likely due to the high utilization of the chain causing many stressed systems to fall behind and fail to gossip those signage points. 

What I recommend is:
1. stop chia
2. delete the peers.dat file at `~\.chia\mainnet\db\peers.dat` (this will force chia to find new peers)
3. start chia

You will likely see those peer transaction queue full logs again (we are looking into reducing the noise from this and making it less spammy) but this should help with the missing signage point issues 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
